tests/test_web/test_api_v1.py

```diff
@@ -1316,3 +1316,146 @@ def test_regenerate_bundle_batch_invalid_input(payload, error_msg, app, auth_env
 
     assert rv.status_code == 400, rv.json
     assert rv.json == {'error': error_msg}
+
+
+@mock.patch('iib.web.api_v1.handle_add_request')
+@mock.patch('iib.web.api_v1.handle_rm_request')
+@mock.patch('iib.web.api_v1.messaging.send_messages_for_new_batch_of_requests')
+def test_add_rm_batch_success(mock_smfnbor, mock_hrr, mock_har, app, auth_env, client, db):
+    annotations = {'msdhoni': 'The best captain ever!'}
+    data = {
+        'annotations': annotations,
+        'build_requests': [
+            {
+                'bundles': ['registry-proxy/rh-osbs/lgallett-bundle:v1.0-9'],
+                'binary_image': 'registry-proxy/rh-osbs/openshift-ose-operator-registry:v4.5',
+                'from_index': 'registry-proxy/rh-osbs-stage/iib:v4.5',
+                'add_arches': ['amd64'],
+                'cnr_token': 'no_tom_brady_anymore',
```

Comment:
:cry:

Reply:
haha, I was hoping you would notice it :p